doc/whats_new/v0.20.rst

```diff
@@ -354,6 +354,12 @@ Metrics
   :func:`mutual_info_score`.
   :issue:`9772` by :user:`Kumar Ashutosh <thechargedneutron>`.
 
+- Fixed a bug in :func:`metrics.cluster.fowlkes_mallows_score` to avoid integer
+  overflow. Casted return value of `contingency_matrix` to `int64` and computed
+  product of square roots rather than square root of product.
+  :issue:`9515` by :user:`Alan Liddell <aliddell>` and
+  :user:`Kumar Ashutosh <thechargedneutron>` and :user:`Manh Dao <manhdao>`.
```

Comment:
Did you use their work here?

Reply:
The tests were Kumar's and the return value comes from Manh's suggestion in #9515.

Reply:
Kumar's tests are credited elsewhere I think. Thanks for being careful about this

Reply:
Should I remove Kumar's name from this?